src/bosh-director/spec/unit/starter_spec.rb

```diff
@@ -127,16 +136,17 @@ module Bosh::Director
               instance: instance,
               agent_client: agent_client,
               update_config: update_config,
+              task: task,
             )
           end.to raise_exception(Bosh::Director::AgentJobNotRunning)
         end
       end
 
       context 'when the task is cancelled' do
         it 'should stop execution if task was canceled' do
-          task = Bosh::Director::Models::Task.make(id: 42, state: 'cancelling')
+          t = Bosh::Director::Models::Task.make(id: 42, state: 'cancelling')
```

Comment:
Minor quibble, but I much prefer the full variable name. I think it's the pattern we keep throughout most of the codebase but this is such a minor test with a pretty obvious output so 🤷

Reply:
The variable was renamed because of a naming conflict with this [`task`](https://github.com/cloudfoundry/bosh/pull/2266/files#diff-8a563477d5608c319453197e0bd572ecR7) which is of type `Bosh::Director::EventLog::Task`. Using a more descriptive variable name which would not be `task` would have force me to split the statement over multiple lines due to the rubocop max line length limitation. As you said yourself the scope of this test is limited, I chose to go with the short variable name.